### PR TITLE
Add initial materials

### PR DESCRIPTION
**Pull request by JacobValdemar**
Any thoughts on the content?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,58 @@
 # Learning materials
-This repository contains learning materials, that is relevant for new engineers at Lunar.
+
+This repository contains learning materials, that are relevant for new engineers at Lunar.
```
Comment (Crevil) [inline README.md]:
```suggestion
This repository contains learning materials, that are relevant for engineers at Lunar.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,58 @@
...
+## Content
+
+1. Microservices
+2. Cloud Native
+3. Kubernetes and Containers
+4. Golang
+5. CI/CD and GitOps
+6. Observability
+7. Culture
+8. Other things
```
Comment (Crevil) [inline README.md]: Markdown lists are magical in that if you use `1.` for all items you don't need to update it when you add new items.

```suggestion
1. Microservices
1. Cloud Native
1. Kubernetes and Containers
1. Golang
1. CI/CD and GitOps
1. Observability
1. Culture
1. Other things
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,58 @@
...
+
+**Go by Example**. https://gobyexample.com. (Primarily for reference)
+
+## CI/CD and GitOps
+
+**GitOps - Operations by Pull Request** by Kasper Nissen. https://tech.lunar.app/blog/2020-03-10-gitops-operations-by-pull-request.
+
+**Release Manager** by Lunar. https://github.com/lunarway/release-manager.
+
+**Shuttle** by Lunar. https://github.com/lunarway/shuttle.
```
Comment (Crevil) [inline README.md]: I don't really find these learning materials TBH. They are very specific tools (and not greatly documented as such). I think we should iam for more general articles on these matters.

**Review by Crevil — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (JacobValdemar) [reply README.md]: Alright 👍

Comment (JacobValdemar) [reply README.md]: https://github.com/lunarway/learning-materials/pull/3/commits/baf64ff5713f7554a3ed6911af4b0e8b3f5d13d9

**Pull request merged**